src/uwtools/drivers/orog.py

```diff
@@ -82,6 +83,22 @@ def provisioned_rundir(self):
             self.runscript(),
         ]
 
+    @task
+    def runscript(self):
+        """
+        The runscript.
+        """
+        path = self._runscript_path
+        yield self.taskname(path.name)
+        yield asset(path, path.is_file)
+        yield None
+        envvars = {
+            "KMP_AFFINITY": "disabled",
+            "OMP_NUM_THREADS": self.config.get(STR.execution, {}).get(STR.threads, 1),
+            "OMP_STACKSIZE": "2048m",
+        }
+        self._write_runscript(path=path, envvars=envvars)
```

Comment:
This program is OMP-enabled, but not MPI. It needed a few extras in its runscript.